app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -0,0 +1,416 @@
+<%
+  title ||= "GOV.UK - The best place to find government services and information"
+  html_lang ||= "en"
+%>
+
+<!DOCTYPE html>
+  <!--[if lt IE 9]><html class="lte-ie8" lang="<%= html_lang %>"><![endif]-->
+  <!--[if gt IE 8]><!--><html lang="<%= html_lang %>"><!--<![endif]-->
+  <head>
+    <meta charset="utf-8" />
+    <title><%= title %></title>
+
+    <%= csrf_meta_tags %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'print', media: 'print' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+
+    <%= stylesheet_link_tag "fonts.css", media: "all", integrity: true, crossorigin: "anonymous" %>
+    <!--[if lt IE 9]><%= javascript_include_tag "ie.js", integrity: true, crossorigin: "anonymous" %><![endif]-->
+
+    <%# the colour used for theme-color is the standard palette $black from
+        https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/colours/_palette.scss %>
+    <meta name="theme-color" content="#0b0c0c" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <%# The default og:image is added below :head so that scrapers see any custom metatags first, and this is just a fallback %>
+    <meta property="og:image" content="<%= asset_path "opengraph-image.png" %>">
```

Comment:
Have we moved this asset into the gem?

Reply:
We're pulling the assets from GOVUK Frontend 👍

Reply:
Thanks for clarifying @danacotoran - do we need to rename? I can see `govuk-opengraph-image.png` in the GOVUK Frontend repo

Reply:
@vanitabarrett I think you're looking at the outdated version of `_layout_for_public.html.erb`.
Looks like we renamed this to call for `govuk-opengraph-image.png` at some point  🤔https://github.com/alphagov/govuk_publishing_components/blob/3d67d5a36428168100207e0d441834c280e2d6de/app/views/govuk_publishing_components/components/_layout_for_public.html.erb#L34

Reply:
D'oh, makes sense! Thanks 🙂